lib/trax/model/attributes/types/string.rb

```diff
@@ -24,6 +27,18 @@ def initialize(*args, target_klass:)
 
               @target_klass = target_klass
             end
+
+            def type_cast_from_user(value)
```

Comment:
its a rails typecasting interface method name, it gets called anytime its being set but not out of the database

http://www.rubydoc.info/github/rails/rails/ActiveRecord%2FType%2FValue%3Atype_cast_from_user